src/main/config/config.c

```diff
@@ -945,21 +981,21 @@ bool feature(uint32_t mask)
 
 void featureSet(uint32_t mask)
```

Comment:
This method is also used by gps.c and rx.c and should likely be changed to use the new method above.